src/svgcontext.ts

```diff
@@ -342,7 +358,12 @@ export class SVGContext extends RenderContext {
       if (attrNamesToIgnore && attrNamesToIgnore[attrName]) {
         continue;
       }
-      element.setAttributeNS(null, attrName, attributes[attrName]);
+      if (
+        attributes[attrName] &&
+        (this.groupAttributes.length == 0 ||
+          attributes[attrName] != this.groupAttributes[this.groupAttributes.length - 1][attrName])
```

Comment:
Only apply attribute if it is not already defined or it has a different value in the stack of groups.